### DISCUSSION TITLE
Make dashboard header sticky

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#409] [Anything Else]
"When I click on a link in the dashboard to filter the analytics, I have to scroll back up to reset it. It would be nice if that reset button was floating?"

(feedback via email)

**Comment by IndigoBee** [comment]
yes, or be able to click a second time on a filter to unfilter.

Example:
I want to click again on "new event" to unselect it
![image](https://user-images.githubusercontent.com/33088396/98935095-9805f400-2515-11eb-9e5f-f5def444af01.png)

**Comment by Vigasaurus** [comment]
This is what I was able to come up with to achieve both this, and the same for selecting time ranges, while still allowing some user choice for screen real estate usage. I'd love to hear feedback on both the the UI and UX of it all before I PR. 

![2020-12-14_05-25-32](https://user-images.githubusercontent.com/19434157/102077104-63b07c80-3dce-11eb-9836-4fdad70fd5d0.gif)

**Reply by ukutaht** [reply]
Looks good!  Personally I don't think the switch is needed, the header can just always be sticky.

Maybe a little border or shadow underneath the floating header to separate it from the rest of the page?

**Reply by Vigasaurus** [reply]
Yeah can do. Only reason I thought about the option to unpin it was for small screens and multiple filters that stack vertically (think mobile) constantly taking anywhere from 10-40% of the screen space

**Comment by metmarkosaric** [comment]
This is now live!

- See the chosen time period and switch dates
-  See the chosen filters and clear filters
-  Filter drop-down when using many filters
